CMakeLists.txt

```diff
@@ -15,6 +15,40 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 # User options.
 option(OPENMP "use OpenMP threading" ON)
 option(ENABLE_DOCS "Enable generation of doxygen-based documentation." OFF)
+
+# Utilities to be built (Default: ALL)
+option(ICEBLEND "Enable building emcsfc_ice_blend.fd" ON)
+option(SNOW2MDL "Enable building emcsfc_snow2mdl.fd" ON)
+option(GCYCLE "Enable building global_cycle.fd" ON)
+option(LSM "Enable building lsm_routines.fd" ON)
+option(FRENCTOOLS "Enable building fre-nctools.fd" ON)
+option(GRIDTOOLS "Enable building grid_tools.fd" ON)
+option(CHGRES "Enable building chgres_cube.fd" ON)
+option(OROG_MASK_TOOLS "Enable building orog_mask_tools.fd" ON)
+option(SFC_CLIMO_GEN "Enable building sfc_climo_gen.fd" ON)
+option(VCOORD_GEN "Enable building vcoord_gen.fd" ON)
+option(FVCOMTOOLS "Enable building fvcom_tools.fd" ON)
+option(GBLEVENTS "Enable building gblevents.fd" ON)
+option(CPLD_GRIDGEN "Enable building cpld_gridgen.fd" ON)
+option(WEIGHT_GEN "Enable building weight_gen.fd" ON)
+
+# Option to build application specific utilities
+option(GFS "Enable building GFS-only utilities" OFF)
+
+# When building the GFS, the following need not be built
+if(GFS)
```

Comment:
One odd thing I noticed. If you don't use double quotes, the message prints with no spaces between the words.

Reply:
You are correct @GeorgeGayno-NOAA 
I have replaced them with double quotes in dfb285b